src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -210,7 +210,9 @@
          (launch/render-button {:workspace-id (:workspace-id props)
                                 :config-id {:namespace (config "namespace") :name (config "name")}
                                 :root-entity-type (config "rootEntityType")
-                                :disabled? (:locked? @state)
+                                :disabled? (if (:locked? @state) "This workspace is locked"
+                                             (if (not (:bucket-access? props)) (str "You do not currently have access "
+                                             "to the Google Bucket associated with this workspace") nil))
```

Comment:
This is ripe for a `cond`:

    (cond (:locked? @state) "This workspace is locked"
          (not (:bucket-access? props)) <that text>)

And if all cases fall through it evaluates to `nil`, which is what you want.